tests/macsec/macsec_config_helper.py

```diff
@@ -125,9 +147,12 @@ def cleanup_macsec_configuration(duthost, ctrl_links, profile_name):
     for dut_port, nbr in ctrl_links.items():
         disable_macsec_port(duthost, dut_port)
         disable_macsec_port(nbr["host"], nbr["port"])
-        delete_macsec_profile(nbr["host"], nbr["port"], profile_name)
         devices.add(nbr["host"])
-        delete_macsec_profile(duthost, dut_port, profile_name)
+
+    # Delete the macsec profile once after it is removed from all interfaces. if we pass port as None, 
+    # the profile is removed from the DB in all namespaces.
+    delete_macsec_profile(duthost, None, profile_name)
```

Comment:
I think we should also to delete the profile in neighbor device
`delete_macsec_profile(nbr["host"], None, profile_name)`

Reply:
yes thanks Ze for noting, have updated the logic after the for loop to clean the profile in all the neighbors which is cleaner.